mgmtsystem/rhevm.py

```diff
@@ -95,6 +95,10 @@ def __init__(self, hostname, username, password, **kwargs):
         else:
             url = 'https://%s/api' % hostname
 
+        if 'version' in kwargs:
```

Comment:
There is more to it than this I think, the RHEVM API has moved, we will need to investigate this. Does the old client API support 4.0? We froze the requirement for ovirtsdk to get around it.